### PR TITLE
:art: Event block redesign

### LINKED ISSUE #635: :art:   Event Block Redesign: block redesign
## User stories

As a user, I want the "Event Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/event-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/event-block/`.

### PR DESCRIPTION
**Pull request by darlene9497**
# Description

This pull request addresses the redesign of the "Event Block" component as outlined in the provided designs. The goal of this PR is to enhance the usability and visual appeal of the component while adhering to the acceptance criteria,

Fixes #635

# Screenshot 
Screenshot from 2023-10-13 12-18-20
Screenshot from 2023-10-13 12-18-33

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/event-block.jsplumb.ts [hunk 1]
```diff
@@ -21,8 +21,16 @@ export function _EventBlockDecoratePlumb(block: EventBlock, comp: ComponentRef<B
...
+    anchor: [0, 0.2, 0, 1],
+    maxConnections: -1
+  });
+
+  jsPlumb.addEndpoint(comp.location.nativeElement, {
+    source: true,
+    cssClass:"block_endpoint",
+    endpoint: "Dot",
+    anchor: [1, 0.75, 0, 1],
     maxConnections: -1
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/event-block.jsplumb.ts:34]: Use the default option but style it no need for the extra addEndpoint call

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

**Comment by darlene9497** [comment]
@LemmyMwaura I have made the requested changes

**Comment by LemmyMwaura** [comment]
Kindly resolve your conflicts @darlene9497

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/event-block.jsplumb.ts [hunk 2]
```diff
@@ -21,9 +21,8 @@ export function _EventBlockDecoratePlumb(block: EventBlock, comp: ComponentRef<B
     // Whether the anchor is target (Other Block -> This Block)
     target: true,
     cssClass: 'block_endpoint',
-    endpoint: 'Rectangle',
-    anchor: "Left",
-    maxConnections: -1
+    endpoint: 'Dot',
+    anchor: [0, 0.18, 0, 1]
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/event-block.jsplumb.ts:25]: leave the anchor as left for now and max connections should still be set to -1.

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**